### ISSUE TITLE
Create subgraph.

### ISSUE THREAD

**Issue by deepesh-kn** [OpenST/mosaic-chains#86]
Create subgraph (graph init) done once manually in ticket and commit files to save in graph folder under
- origin_chains/<chain_name>/graph (“ethereum” and “ropsten”)
- auxiliary_chains/<chain_id>/graph (1414, 1407, 1406)
- It needs contracts ABI file path

**Comment by puneet-khushwani-eth** [comment]
@sarvesh-ost , while running `deploy local` locally today we got these 

`in src/EIP20GatewayMapping.ts(113,8)
WARNING AS206: Compiling constant with non-constant initializer as mutable.
  const entity = new RevertRedeemComplete(
   `${event.transaction.hash.toHex()}-${event.logIndex.toString()}`,
 );`

I guess this has something to do with the `` operation we have used. 

Also after ingesting some data, we see that id is not processed correctly. Instead of seeing `0x232-323` we see `${event.transaction.hash.toHex()}-${event.logIndex.toString()}` as ids. 

Changing `${event.transaction.hash.toHex()}-${event.logIndex.toString()}` to `event.transaction.hash.toHex() + "-" + event.logIndex.toString()` fixes this.
